nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-client-service-api/src/main/java/org/apache/nifi/mongodb/MongoDBClientService.java

```diff
@@ -44,6 +44,22 @@
             .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
             .addValidator(StandardValidators.URI_VALIDATOR)
             .build();
+     PropertyDescriptor DB_USER = new PropertyDescriptor.Builder()
+            .name("Database User")
```

Comment:
Please use both ``.name()`` and ``.displayName()``

Reply:
I did copy paste from DBCPConnectionPool, so we need to change there too. But I'll not do this in this PR to not mess.